pkg/vacuum/vacuum.go

```diff
@@ -220,5 +254,8 @@ func (e *Engine) worker(ctx context.Context, id int, todo <-chan string) {
 			log.Error("msg", "failed to vacuum chunk", "chunk", chunk, "worker", id, "error", err)
 			// don't return error here. attempt to vacuum other chunks. keep working
 		}
+		tablesVacuumedTotal.Inc()
+		tablesNeedingVacuum.Dec()
```

Comment:
I am torn on this decrement. On one hand its true you just vacummed a table, on the other: 
1) its really weird when against the 1000 limit
2) another table may have been created that needs vacumming

Overall I think there is something nice about this gauge being always set from data in the DB. The "single source of truth"

but not sure and dont feel super strongly

Reply:
okay. I'll update the help of the gauge to hopefully better describe it